app/views/legislation/processes/_process.html.erb

```diff
@@ -21,43 +21,43 @@
     <% column_width = 12 / process.enabled_phases_and_publications_count %>
     <div class="column row">
       <div class="small-12 column legislation-calendar-info">
-        <p><%= t('legislation.processes.shared.key_dates') %></p>
+        <p><%= t("legislation.processes.shared.key_dates") %></p>
       </div>
     </div>
 
     <div class="column row small-collapse medium-uncollapse legislation-calendar">
       <% if process.debate_phase.enabled? %>
         <div class="small-6 medium-<%= column_width %> column">
-            <h5><%= t('legislation.processes.shared.debate_dates') %></h5>
-            <p><%= format_date(process.debate_start_date) %> - <%= format_date(process.debate_end_date) %></p>
+          <h4><%= t("legislation.processes.shared.debate_dates") %></h4>
+          <p><%= format_date(process.debate_start_date) %> - <%= format_date(process.debate_end_date) %></p>
         </div>
       <% end %>
 
       <% if process.draft_publication.enabled? %>
         <div class="small-6 medium-<%= column_width %> column">
-            <h5><%= t('legislation.processes.shared.draft_publication_date') %></h5>
-            <p><%= format_date(process.draft_publication_date) %></p>
+          <h4><%= t("legislation.processes.shared.draft_publication_date") %></h4>
+          <p><%= format_date(process.draft_publication_date) %></p>
         </div>
       <% end %>
 
       <% if process.proposals_phase.enabled? %>
         <div class="small-6 medium-<%= column_width %> column">
-            <h5><%= t('legislation.processes.shared.proposals_dates') %></h5>
-            <p><%= format_date(process.proposals_phase_start_date) %> - <%= format_date(process.proposals_phase_end_date) %></p>
+          <h4><%= t("legislation.processes.shared.proposals_dates") %></h4>
+          <p><%= format_date(process.proposals_phase_start_date) %> - <%= format_date(process.proposals_phase_end_date) %></p>
         </div>
       <% end %>
 
       <% if process.allegations_phase.enabled? %>
         <div class="small-6 medium-<%= column_width %> column">
-            <h5><%= t('legislation.processes.shared.allegations_dates') %></h5>
-            <p><%= format_date(process.allegations_start_date) %> - <%= format_date(process.allegations_end_date) %></p>
+          <h4><%= t("legislation.processes.shared.allegations_dates") %></h4>
+          <p><%= format_date(process.allegations_start_date) %> - <%= format_date(process.allegations_end_date) %></p>
         </div>
       <% end %>
 
       <% if process.result_publication.enabled? %>
         <div class="small-6 medium-<%= column_width %> column">
-            <h5><%= t('legislation.processes.shared.result_publication_date') %></h5>
-            <p><%= format_date(process.result_publication_date) %></p>
+          <h4><%= t("legislation.processes.shared.result_publication_date") %></h4>
```

Comment:
I think in next reunion we should talk about this... "Should we use double or single quotes on I18n.t/l calls". I don't have a strong opinion about it, although I feel the urge to use single quotes but I see that the convention in the codebase is using double quotes because its normal use is in html.erb files that use double quotes for everything.

Reply:
+1 to choose one style. I just have changed it to be consistent with the rest of code 😉

Reply:
Should we create an issue about it or you'll just mention it along your opinion on next meeting?

Reply:
I would create the issue to unify all i18n keys after deciding the format to use 😉